django/db/backends/sqlite3/schema.py

```diff
@@ -379,3 +390,7 @@ def add_constraint(self, model, constraint):
 
     def remove_constraint(self, model, constraint):
         self._remake_table(model, remove_constraint=constraint)
+
+    @cached_property
+    def sqlite_version(self):
+        return sqlite_version_info
```

Comment:
We usually define these on `DatabaseWrapper` objects, that is `django.db.backends.sqlite3.base.DatabaseWrapper`. You'll be able to refer to it using `self.connection.sqlite_version`.

Reply:
I'll fix!